src/middleware/get-payload.ts

```diff
@@ -1,19 +1,19 @@
-module.exports = getPayload;
+import { IncomingMessage } from "http";
+import { Payload } from "../types";
 
-function getPayload(request) {
+export function getPayload(request: IncomingMessage): Promise<Payload> {
   // If request.body already exists we can stop here
   // See https://github.com/octokit/webhooks.js/pull/23
-  if (request.body) {
-    return Promise.resolve(request.body);
-  }
+  // if (request.body) {
+  //   return Promise.resolve(request.body);
+  // }
```

Comment:
@gr2m with latest typing changes I had to disable this lines because `request.body` is not an existing property.

Do you have more context on what was the issue in this scenario? https://github.com/octokit/webhooks.js/pull/23

Reply:
I think in serverless environments such as AWS lambda, `request.body` is already present. And if it is, we can directly return it, instead of trying to retrieve it again, which will fail

Reply:
> I think in serverless environments such as AWS lambda, `request.body` is already present. And if it is, we can directly return it, instead of trying to retrieve it again, which will fail

I see. Do you have any idea of how we should give support to this and at the same time be `@types/http` compliant? 

Checking which I think is the official [aws-lambda](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/aws-lambda) type definition apparently is not extending from `@types/http.IncomingMessage`

Reply:
What about 
```typescript
export function getPayload(request: IncomingMessage & { body?: Payload }): Promise<Payload> {
```

too hacky?